src/implementations/twig/components/card/card.story.js

```diff
@@ -193,9 +207,23 @@ export default {
   decorators: [withCode, withNotes],
 };
 
-export const Card = (args) => card(prepareData(dataCard, args));
+export const Default = (args) => card(prepareData(dataCardDefault, args));
+
```

Comment:
At this point i wonder about the reason why having multiple stories in card. I find that confusing considering that all the optional content of a card has toggles to show/hide them, i think that one single story "default" would be enough, but you probably based also on requests by comm, so just a personal thought..

Reply:
I won't say that this is a "request", but all their design mention 3 examples of cards (default, tags, list), so I guess that having them with the same naming would avoid some discussions